### DISCUSSION TITLE
How to default set course filter when we search a content in mobile app?

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#266] [General Discussion]
While searching any content in mobile app, there is no default filter set to show the results.

**Scenario:** 
While searching any content in mobile app, I want to show the results of course by default. In filters (sb-pill) course should be selected by default.
How can I achieve this?

cc: @swayangjit @vpPavithra @luckynvdu @baljeetsingh

**Comment by Krishnaj20** [comment]
@nidhikachhawa24 - the global search shows all matching categories of content/collection/question sets (courses, textbooks etc). User can then filter out a specific category by applying filters in the search results page. This is similar to how linkedin learning has implemented their search (functionally).  Would like to understand why you would want to only show courses in the search and not relevant content?

**Reply by nidhikachhawa24** [reply]
@Krishnaj20 
It is a requirement of tenant - as they are mostly going to create course and want to show only course by default.

So just wanted to understand whether we can customise the global search or not?

**Reply by swayangjit** [reply]
@nidhikachhawa24 There are multiple pages where we apply content search filter. If you want to apply only in global search then you have to update the following form
`{
  "request": {
    "type": "config",
    "subType": "content_v2",
    "action": "filter"
  }
}`
and add only Course
